pycbc/frame/frame.py

```diff
@@ -300,10 +284,8 @@ def frame_paths(frame_type, start_time, end_time, server=None, url_type='file'):
     >>> paths = frame_paths('H1_LDAS_C02_L2', 968995968, 968995968+2048)
     """
     site = frame_type[0]
-    connection = datafind_connection(server)
-    connection.find_times(site, frame_type,
```

Comment:
I cant see why find_times _was_ called (not a showstopper), but it would be good to know why - this was in [this commit](https://github.com/gwastro/pycbc/commit/a138284c4874d763222f803db1052adc433f1042), which doesn't say why

It would be good to know if e.g. this function provides a better error message

Reply:
I can't see why this was called either. According to the documentation it doesn't do anything, and didn't do anything ... Keeping it would add a small slowdown, so it is removed.